src/main.rs

```diff
@@ -19,64 +21,67 @@ use cargo_binstall::{
     *,
 };
 
-#[derive(Debug, StructOpt)]
+static DEFAULT_TARGET: OnceCell<String> = OnceCell::new();
+
+#[derive(Debug, Parser)]
+#[clap(version, about = "Install a Rust binary... from binaries!")]
 struct Options {
     /// Package name or URL for installation
     /// This must be either a crates.io package name or github or gitlab url
-    #[structopt()]
+    #[clap(value_name = "crate")]
     name: String,
 
     /// Filter for package version to install, in Cargo.toml format.
     /// Use `=1.2.3` to install a specific version.
-    #[structopt(long, default_value = "*")]
+    #[clap(long, default_value = "*")]
     version: String,
 
     /// Override binary target, ignoring compiled version
-    #[structopt(long, default_value = TARGET)]
+    #[clap(help_heading = "OVERRIDES", long, default_value = DEFAULT_TARGET.get().unwrap())]
     target: String,
```

Comment:
Is it better to use `Option<String>` so that we don't have to define a global variable `DEFAULT_TARGET`?

Reply:
The "cool thing" about the way I've got it here is that the default shows up in the `--help`, and I haven't figured out a cleaner way to do that with the clap derive API.

Reply:
Oh ok, that makes sense now.